featurebyte/api/item_view.py

```diff
@@ -303,8 +303,9 @@ def _are_columns_derived_only_from_event_table(self, column_names: List[str]) ->
         -------
         bool
         """
+        # FIXME: This method is may not work correctly when the column is from SCD table joined with EventTable
         operation_structure = self.graph.extract_operation_structure(
-            self.node, keep_all_source_columns=True
+            self.node, keep_all_source_columns=False
```

Comment:
This addresses the comment in https://github.com/featurebyte/featurebyte/pull/1524#discussion_r1263359589.